ui/pages/notifications/notification-components/feature-announcement/feature-announcement.tsx

```diff
@@ -117,5 +117,17 @@ export const components: NotificationComponent<FeatureAnnouncementNotification>
             isExternal={true}
           />
         ) : null,
+      ExternalLink: ({ notification }) =>
+        notification.data.externalLink ? (
+          <NotificationDetailButton
+            notification={notification}
+            variant={ButtonVariant.Secondary}
+            text={notification.data.externalLink.externalLinkText}
+            href={`/${notification.data.externalLink.externalLinkUrl}`}
```

Comment:
I don't know if this href is correct for external links.

For external links, we want to navigate them out of the extension. Ideally in a new window right?

Reply:
Since it currently is `/<external link url>`, you will end up with `chrome:extension://www.google.com`

Reply:
Good catch! Bad copy/paste. Fixed @Prithpal-Sooriya